sources/HUBViewControllerDefaultScrollHandler.m

```diff
@@ -68,6 +69,26 @@ - (CGPoint)targetContentOffsetForEndedScrollInViewController:(UIViewController<H
     return proposedContentOffset;
 }
 
+- (CGPoint)contentOffsetForDisplayingComponentAtIndex:(NSUInteger)componentIndex
+                                       scrollPosition:(UICollectionViewScrollPosition)scrollPosition
+                                       viewController:(UIViewController<HUBViewController> *)viewController
+{
+    CGRect const componentFrame = [viewController frameForBodyComponentAtIndex:componentIndex];
```

Comment:
Really cuirous as to why this should be the responsibility of the scroll handler. Seems like something the view controller should do.